### PR TITLE
wine-2.0.2_3: port changes from sane-backends-1.0.27_3

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
Could you use the same PR (same revision bump) to updated mpg123 to latest version (1.25.5) too? Thanks.

**Comment by yurikoles** [comment]
@fxcoudert done

File: Formula/wine.rb [hunk 1]
```diff
@@ -417,6 +417,10 @@ def install
                                 "--enable-local-backends",
                                 "--with-usb=yes",
                                 *depflags
+          # Remove for > 1.0.27
+          # Workaround for bug in Makefile.am described here: https://goo.gl/vqzVk5.
+          # It's already fixed in commit 519ff57. See https://goo.gl/Cmp6Br.
```
Comment (JCount) [inline Formula/wine.rb]: Please don't use URL shortening in the formula, even in a comment.

**Review by JCount — CHANGES_REQUESTED** [review]
The commit message should be in the standard form. Something like `wine: workaround for sane-backends Makefile bug, update mpg123`.

**Comment by yurikoles** [comment]
@JCount done

**Review by fxcoudert — APPROVED** [review]

**Review by JCount — APPROVED** [review]

**Comment by JCount** [comment]
@yurikoles Thank you for the contribution to Homebrew!! 🎆

**Pull request closed without merging**